### ISSUE TITLE
Stub README.md remains alongside active README.MD

### ISSUE THREAD

**Issue by gjsjohnmurray** [microsoft/vsmarketplace#9]
I think using lowercase file extension is more conventional.

**Comment by isidorn** [comment]
@gjsjohnmurray where do you see this? Just to clarify...

**Comment by gjsjohnmurray** [comment]
![Screenshot_20210825-150552](https://user-images.githubusercontent.com/6726799/130805776-de223493-c905-4bc5-93de-c905f59f6bae.png)

In this repo's code. See bottom of the screenshot above.

**Comment by isidorn** [comment]
Yeah, good point. Let's fix this. @gjsjohnmurray would you like to provide a PR?
If not I can also change this in case @prashantvc agrees
